app/tests/unit/components/Form/ProjectMilestoneReportFormSummary.test.tsx

```diff
@@ -185,56 +220,56 @@ describe("The Project Milestone Report Form Summary", () => {
     expect(screen.getByText(/\$88\.00/i)).toBeInTheDocument();
   });
 
-  const latestCommittedData = {
-    latestCommittedMilestoneFormChanges: {
-      edges: [
-        {
-          node: {
-            newFormData: {
-              totalEligibleExpenses: 1000,
-              description: "charmander",
-              projectId: 1,
-              reportingRequirementIndex: 1,
-              reportType: "General",
-              reportDueDate: "2020-01-10T23:59:59.999-07:00",
-              reportingRequirementId: 1,
-              hasExpenses: true,
-              calculatedGrossAmount: 567,
-              calculatedNetAmount: 789,
-              calculatedHoldbackAmount: 891,
-              adjustedNetAmount: 89,
-              adjustedGrossAmount: 67,
-              adjustedHoldbackAmount: 91,
+  it("Displays diffs of the data fields that were updated and shows latest committed values", () => {
```

Comment:
This one and the following one are just moving variables around? (Hard to see if there's a real diff in here)

Reply:
You're right this one is hard to follow. Just to clarify, this commit is all about moving variables around to make the test a bit more readable and close to what we do in other tests: https://github.com/bcgov/cas-cif/pull/1805/commits/113b773f0c1fc0697f7a80f7403dc9ccf5c74b60